manifests/cluster-network-addons/0.4.0/cluster-network-addons-operator.0.4.0.clusterserviceversion.yaml

```diff
@@ -151,6 +151,8 @@ spec:
                     value: docker.io/nfvpe/multus:latest
                   - name: LINUX_BRIDGE_IMAGE
                     value: quay.io/kubevirt/cni-default-plugins:latest
+                  - name: BRIDGE_MARKER_IMAGE
```

Comment:
there should be no changes under 0.4.0. this directory is automatically generated when the operator is released.

Reply:
Please edit it in `Makefile`, `tools/manifests-templator/manifests-templator.go` and `pkg/components/components.go`. Also, please do release bridge-marker on u/s (with a tag) and create a marker image with the tag. We should not link :latest here

Reply:
Removed the files from 0.4.0